pkg/network/infraconfigurators/bridge.go

```diff
@@ -265,10 +292,19 @@ func (b *BridgePodNetworkConfigurator) switchPodInterfaceWithDummy() error {
 	// Replace original pod interface IP address to the dummy
 	// Since the dummy is not connected to anything, it should not affect networking
 	// Replace will add if ip doesn't exist or modify the ip
-	err = b.handler.AddrReplace(dummy, &b.podIfaceIP)
-	if err != nil {
-		log.Log.Reason(err).Errorf("failed to replace original IP address to dummy interface: %s", originalPodInterfaceName)
-		return err
+	if b.podIfaceIP != nil {
```

Comment:
again, I think this is redundant: this dummy interface only exists when there is an IPv4 address in the pod interface.

Reply:
Same as above, unless I'm missing something, this could be `nil` for IPv6-only cases.